### PR TITLE
overrides: fasttrack ignition-2.2.1-5.git2d3ff58.fc31

### PR DESCRIPTION
**Pull request by dustymabe**
This gets us new networking improvements added in https://github.com/coreos/ignition-dracut/pull/174.

### CODE REVIEW

**Review by jlebon — APPROVED** [review]

**Comment by dustymabe** [comment]
CI failed, but I don't understand why:

```
+ set -xeuo pipefail
+ cd /srv/fcos
+ cosa fetch --strict
info: Missing CAP_SYS_ADMIN; using virt
Formatting 'cache/cache2.qcow2', fmt=qcow2 size=10737418240 cluster_size=65536 lazy_refcounts=off refcount_bits=16
Config commit: 8bd590146d855717cb07c814b1a26439674259da
Using manifest: /srv/fcos/src/config/manifest.yaml
Running: rpm-ostree compose tree --repo=/srv/fcos/tmp/repo --cachedir=/srv/fcos/cache --touch-if-changed /srv/fcos/tmp/build/tmp/treecompose.changed --unified-core /srv/fcos/src/config/manifest.yaml --download-only --ex-lockfile=/srv/fcos/src/config/manifest-lock.x86_64.json --ex-lockfile=/srv/fcos/src/config/manifest-lock.overrides.x86_64.yaml --ex-lockfile-strict
supermin: warning: /boot/System.map-5.5.17-200.fc31.x86_64: Permission denied (ignored)
Some distro files are not public readable, so supermin cannot copy them
into the appliance.  This is a problem with your Linux distro.  Please ask
your distro to stop doing pointless security by obscurity.
You can ignore these warnings.  You *do not* need to use sudo.
supermin: warning: /boot/initramfs-5.5.17-200.fc31.x86_64.img: Permission denied (ignored)
supermin: warning: /usr/libexec/utempter/utempter: Permission denied (ignored)
supermin: warning: /usr/sbin/unix_update: Permission denied (ignored)
supermin: warning: /var/lib/systemd/random-seed: Permission denied (ignored)
tail: cannot open '/srv/fcos/tmp/runvm-cmd-output.txt' for reading: No such file or directory
tail: '/srv/fcos/tmp/runvm-cmd-output.txt' has appeared;  following new file
Loaded lockfiles:
  /srv/fcos/src/config/manifest-lock.x86_64.json
  /srv/fcos/src/config/manifest-lock.overrides.x86_64.yaml
RPM-OSTree Version: 2020.1
No previous commit for fedora/x86_64/coreos/testing-devel
Enabled rpm-md repositories: fedora-coreos-pool fedora fedora-updates fedora-modular fedora-updates-modular
Updating metadata for 'fedora-coreos-pool'...done
rpm-md repo 'fedora-coreos-pool'; generated: 2020-04-20T21:21:14Z
Updating metadata for 'fedora'...done
rpm-md repo 'fedora'; generated: 2019-10-23T22:52:47Z
Updating metadata for 'fedora-updates'...done
rpm-md repo 'fedora-updates'; generated: 2020-04-27T02:36:17Z
Updating metadata for 'fedora-modular'...done
rpm-md repo 'fedora-modular'; generated: 2019-10-23T22:53:13Z
Updating metadata for 'fedora-updates-modular'...done
rpm-md repo 'fedora-updates-modular'; generated: 2020-04-25T05:28:19Z
script returned exit code 1
```

**Comment by dustymabe** [comment]
The error is error is: 

```
Couldn't find locked package 'ignition-2.2.1-5.git2d3ff58.fc31.x86_64' (pkgs matching NEVRA: 0; mismatched checksums: 0)
```

This is because `cosa fetch --strict` isn't considering the overrides/rpm rpms (fix incoming for that). The other issue is that the error message doesn't show up in the output in CI.

**Comment by dustymabe** [comment]
rebased on top of latest master

**Comment by dustymabe** [comment]
with https://github.com/coreos/coreos-assembler/pull/1404 `cosa fetch --strict --with-cosa-overrides` works

**Comment by jlebon** [comment]
Do you want to try adding `--with-cosa-overrides` to `.cci.jenkinsfile` in this PR?

**Comment by dustymabe** [comment]
> Do you want to try adding `--with-cosa-overrides` to `.cci.jenkinsfile` in this PR?

Does that mean we are no longer using `fcosBuild`?

**Comment by dustymabe** [comment]
ok `extraFetchArgs` was added in https://github.com/coreos/coreos-ci-lib/pull/26. I just pushed a commit to use it.

File: .cci.jenkinsfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ cosaPod {
         cosa buildprep https://builds.coreos.fedoraproject.org/prod/streams/${env.CHANGE_TARGET}/builds
     """)
 
-    fcosBuild(skipInit: true)
+    fcosBuild(make: true, skipKola: true, extraFetchArgs: '--with-cosa-overrides')
```
Comment (jlebon) [inline .cci.jenkinsfile]: Hmm, why are we doing `make: true` and `skipKola: true` here and dropping `skipInit: true`? Was that by mistake?

Comment (dustymabe) [reply .cci.jenkinsfile]: oops.. yeah. copy/paste error.

Comment (dustymabe) [reply .cci.jenkinsfile]: fixed!

**Review by jlebon — APPROVED** [review]
LGTM!

**Pull request merged**